content/en/docs/concepts/containers/images.md

```diff
@@ -228,7 +228,7 @@ Pods can only reference image pull secrets in their own namespace,
 so this process needs to be done one time per namespace.
 {{< /note >}}
 
-#### Referring to an imagePullSecrets on a Pod
+#### Referring to an imagePullSecrets on a pod
```

Comment:
```suggestion
#### Referring to an imagePullSecrets on a Pod
```
Headings are sentence case, but in sentences we write resource names like Pod in UpperCamelCase.